sdk/eventhub/Azure.Messaging.EventHubs/src/EventHubClient.cs

```diff
@@ -271,6 +293,15 @@ public virtual EventSender CreateSender(SenderOptions senderOptions = default)
         ///
         public virtual void Close(CancellationToken cancellationToken = default) => CloseAsync(cancellationToken).GetAwaiter().GetResult();
 
+        /// <summary>
+        ///   Performs the task needed to clean up resources used by the <see cref="EventHubClient" />,
+        ///   including ensuring that the client itself has been closed.
+        /// </summary>
+        ///
+        /// <returns>A task to be resolved on when the operation has completed.</returns>
+        ///
+        public virtual async ValueTask DisposeAsync() => await CloseAsync();
```

Comment:
Should this have a `ConfigureAwait(false)` ?

Reply:
Yes.... yes, it should.